### ISSUE TITLE
Run infer_relevance.ipynb in demo2

### ISSUE THREAD

**Issue by JeremyGohBNP** [os-climate/aicoe-osc-demo#150]
I have an issue when I run this notebook. There seems to be a confusion on where packages are called: 

![image](https://user-images.githubusercontent.com/82595185/164427050-d534fa25-17aa-43b4-b503-0775345b20f9.png)

This cell calls TextRelevanceInfer but it fails because it asks for an argument "kpi_df" that exists in the package version in lib64/python3.8/site-packages but is not required anymore in the .py file implemented in the repo in src/models/relevance_infer.py

I guess this has to do with the import at the beginning of the notebook. This is surprising because my colleague Andra does not face the same problem. Should I modify the paths for the import? Thanks

@DaBeIDS @Shreyanand @chauhankaranraj

**Comment by JeremyGohBNP** [comment]
Hi @Shreyanand, thanks for your quick reply. I did just that and it seems to solve this issue. However, I have trouble getting the models now: it does not look like it's accessible through the S3 connection. In infer_relevance.ipynb:

![image](https://user-images.githubusercontent.com/82595185/164501321-f18be65a-c550-4138-a81d-645ce514b022.png)
This cell returns the following error: 
![image](https://user-images.githubusercontent.com/82595185/164501453-282e6bb1-3fcc-4dfb-881f-cbc7575cde24.png)

**Comment by Shreyanand** [comment]
This seems like a problem with the S3 connection, maybe due to a credentials change. @redmikhail how can we make sure @JeremyGohBNP has the right credentials?

**Comment by JeremyGohBNP** [comment]
FYI, I'm using these credentials, as stated in the notebooks

![image](https://user-images.githubusercontent.com/82595185/164508232-d01ee681-17ab-4f88-82db-6419c87ca4ac.png)

**Comment by Shreyanand** [comment]
The  `AWS_ACCESS_KEY_ID` and the `AWS_SECRET_ACCESS_KEY` have to be the right values for the notebook to work.

**Comment by JeremyGohBNP** [comment]
Yes, I'm using the values I was given in my personal credentials.env file, as I've always used

**Comment by HeatherAck** [comment]
Due to a security incident yesterday, developers need to get a new access key by filing an issue to create a request using this link: [https://github.com/os-climate/OS-Climate-Community-Hub/issues/new?assignees=erikerlandson&labels=Bucket-credentials&template=request-credentials-for-an-os-climate-bucket.md&title=Request+credentials+for+an+os-climate+bucket](https://url.emailprotection.link/?bnFTdGIUdM1j5xn_c3e_m1PxDIbOf9R8LVRn3jLpEiE3SMWhb6yB915f6Ye-syC2gTw-LEFGfofVbWRmt8FHmHIGGcGfrrPNWayVIcDPcaRXm9x0_2A1XirnMnc9i38bwx56IZPhqMj7emOGOO0dso2V7SPu8NDV9xB4p4AN4qjgeAslndMpMhUWbMscqZw_MDPpe0Wm5qWFwDtkAfg_wJsc1BgGJtOEY_YgmY1fyGz5S_FmE4Q512odod4iogxSP).  

This link can also be reached by visiting the OS-Climate Community Hub page, clicking on the Issues link, opening a New issue, and selecting the option to request new credentials.

**Comment by JeremyGohBNP** [comment]
@erikerlandson Thanks for the file with the new credentials. I am not sure I got the right credentials though. The file you provided gives the following ones:

S3_LANDING_ENDPOINT=https://s3.us-east-1.amazonaws.com
S3_LANDING_BUCKET=redhat-osc-physical-landing-647521352890
S3_LANDING_ACCESS_KEY=XXXXX
S3_LANDING_SECRET_KEY=XXXXX

I've been trying to get the new keys for:
 
# s3 credentials
S3_ENDPOINT=https://s3.us-east-1.amazonaws.com
S3_BUCKET=ocp-odh-os-demo-s3
AWS_ACCESS_KEY_ID=xxx
AWS_SECRET_ACCESS_KEY=xxx

**Comment by chauhankaranraj** [comment]
> @erikerlandson Thanks for the file with the new credentials. I am not sure I got the right credentials though. The file you provided gives the following ones:
> 
> S3_LANDING_ENDPOINT=https://s3.us-east-1.amazonaws.com S3_LANDING_BUCKET=redhat-osc-physical-landing-647521352890 S3_LANDING_ACCESS_KEY=XXXXX S3_LANDING_SECRET_KEY=XXXXX
> 
> I've been trying to get the new keys for:
> 
> S3_ENDPOINT=https://s3.us-east-1.amazonaws.com S3_BUCKET=ocp-odh-os-demo-s3 AWS_ACCESS_KEY_ID=xxx AWS_SECRET_ACCESS_KEY=xxx

Hey @JeremyGohBNP, as per #131 we're migrating to using the `redhat-osc-physical-landing-647521352890` bucket instead of `ocp-odh-os-demo-s3` for the demo workflow. So I think the credentials should correspond to that bucket. Are you unable to run the notebooks with this setup?

**Comment by JeremyGohBNP** [comment]
@chauhankaranraj Thanks! I did not realize you were migrating towards the physical landing bucket! It is working fine when I set my credentials for this bucket! 

Another question: I have been facing memory issues and realized that when starting my server, I am not always given the options to choose for the size or the memory. Is there something to activate or to do to make that choice become available? Thanks again for your help!
